packages/ui/index.ts

```diff
@@ -0,0 +1 @@
+export type { CardViewSuggestionsFragment } from './graphql/generated/client';
```

Comment:
Not something for this PR, but i dont really like that plugins are requiring the ui package, feels werid

Reply:
Something we can make separate in the future by having a plugin abstraction for things to include instead.